staging/src/github.com/kubeedge/beehive/pkg/core/model/message.go

```diff
@@ -39,6 +39,8 @@ const (
 	ResourceTypeSaAccess            = "serviceaccountaccess"
 	ResourceTypeCSR                 = "certificatesigningrequest"
 
+	UnholdPodUpgrade = "unhold-pod-upgrade"
```

Comment:
I think it can be defined as an operation, and "unhold-upgrade" is also an operation, not a resource type. Although we can treat it as a resource type to perform simple logical judgment, the readability is not very good.